source/guides/getting-started.md

```diff
@@ -875,12 +875,11 @@ describe Web::Controllers::Books::Create do
     end
 
     it 'sets errors attribute accordingly' do
-      action.call(params)
-
-      refute action.params.valid?
+      response = action.call(params)
+      response[0].must_equal 422
 
-      action.errors.for('book.title').wont_be_empty
-      action.errors.for('book.author').wont_be_empty
+      action.errors[:book][:title].must_equal  ['is missing']
```

Comment:
It looks like errors now have to be accessed through the params object, so in this case `action.params.errors[:book][:title]`